<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>widget</class>
 <widget class="QWidget" name="widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>749</width>
    <height>647</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>13</pointsize>
    <weight>50</weight>
    <bold>false</bold>
    <underline>false</underline>
   </font>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="title_label">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
         <weight>75</weight>
         <bold>true</bold>
         <underline>true</underline>
        </font>
       </property>
       <property name="text">
        <string>GÃ©rer clients et abonnements</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="client_title">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
         <weight>75</weight>
         <bold>true</bold>
         <underline>true</underline>
        </font>
       </property>
       <property name="text">
        <string>Client</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="8" column="0">
          <widget class="QLabel" name="country_label">
           <property name="text">
            <string>Pays: </string>
           </property>
          </widget>
         </item>
         <item row="7" column="0">
          <widget class="QLabel" name="city_label">
           <property name="text">
            <string>Code p. + ville.: </string>
           </property>
          </widget>
         </item>
         <item row="8" column="3" colspan="2">
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <spacer name="horizontalSpacer_4">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="reset_button">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>Reset</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="search_button">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>100</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>Chercher</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="save_button">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>100</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>Sauvegarder</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="email_label">
           <property name="text">
            <string>Email: </string>
           </property>
          </widget>
         </item>
         <item row="2" column="4">
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item alignment="Qt::AlignLeft|Qt::AlignVCenter">
            <widget class="QRadioButton" name="male_button">
             <property name="text">
              <string/>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">buttonGroup</string>
             </attribute>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="male_label">
             <property name="text">
              <string>M</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="female_label">
             <property name="text">
              <string>F</string>
             </property>
            </widget>
           </item>
           <item alignment="Qt::AlignRight">
            <widget class="QRadioButton" name="female_button">
             <property name="text">
              <string/>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">buttonGroup</string>
             </attribute>
            </widget>
           </item>
          </layout>
         </item>
         <item row="3" column="4">
          <widget class="QDateEdit" name="birthdate_field">
           <property name="font">
            <font>
             <pointsize>13</pointsize>
             <weight>50</weight>
             <bold>false</bold>
             <underline>false</underline>
            </font>
           </property>
           <property name="calendarPopup">
            <bool>true</bool>
           </property>
           <property name="date">
            <date>
             <year>1900</year>
             <month>1</month>
             <day>1</day>
            </date>
           </property>
          </widget>
         </item>
         <item row="7" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLineEdit" name="city_zip_field">
             <property name="minimumSize">
              <size>
               <width>50</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>75</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>13</pointsize>
               <weight>50</weight>
               <bold>false</bold>
               <underline>false</underline>
              </font>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="city_name_field">
             <property name="minimumSize">
              <size>
               <width>175</width>
               <height>0</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>13</pointsize>
               <weight>50</weight>
               <bold>false</bold>
               <underline>false</underline>
              </font>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="6" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLineEdit" name="street_name_field">
             <property name="minimumSize">
              <size>
               <width>175</width>
               <height>0</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>13</pointsize>
               <weight>50</weight>
               <bold>false</bold>
               <underline>false</underline>
              </font>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="street_nr_field">
             <property name="minimumSize">
              <size>
               <width>50</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>50</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>13</pointsize>
               <weight>50</weight>
               <bold>false</bold>
               <underline>false</underline>
              </font>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="phone_field">
           <property name="font">
            <font>
             <pointsize>13</pointsize>
             <weight>50</weight>
             <bold>false</bold>
             <underline>false</underline>
            </font>
           </property>
          </widget>
         </item>
         <item row="2" column="3">
          <widget class="QLabel" name="sex_label">
           <property name="text">
            <string>Sexe: </string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="empty_label">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLineEdit" name="email_field">
           <property name="font">
            <font>
             <pointsize>13</pointsize>
             <weight>50</weight>
             <bold>false</bold>
             <underline>false</underline>
            </font>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QLabel" name="reduced_price_label">
           <property name="text">
            <string>Tarif rÃ©duit: </string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="phone_label">
           <property name="text">
            <string>Phone: </string>
           </property>
          </widget>
         </item>
         <item row="3" column="3">
          <widget class="QLabel" name="birthdate_label">
           <property name="text">
            <string>Date de naissance: </string>
           </property>
          </widget>
         </item>
         <item row="8" column="1">
          <widget class="QLineEdit" name="country_field">
           <property name="font">
            <font>
             <pointsize>13</pointsize>
             <weight>50</weight>
             <bold>false</bold>
             <underline>false</underline>
            </font>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="street_label">
           <property name="text">
            <string>Rue + nr.:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="4">
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <widget class="QCheckBox" name="reduced_price_checkbox">
             <property name="text">
              <string/>
             </property>
             <property name="tristate">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="first_name_field">
           <property name="font">
            <font>
             <pointsize>13</pointsize>
             <weight>50</weight>
             <bold>false</bold>
             <underline>false</underline>
            </font>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="last_name_field">
           <property name="font">
            <font>
             <pointsize>13</pointsize>
             <weight>50</weight>
             <bold>false</bold>
             <underline>false</underline>
            </font>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="last_name_label">
           <property name="text">
            <string>Nom: </string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="first_name_label">
           <property name="text">
            <string>PrÃ©nom:</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout_3"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="abo_title">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
         <weight>75</weight>
         <bold>true</bold>
         <underline>true</underline>
        </font>
       </property>
       <property name="text">
        <string>Abonnements</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QTabWidget" name="tabWidget">
         <property name="font">
          <font>
           <pointsize>13</pointsize>
           <weight>50</weight>
           <bold>false</bold>
           <underline>false</underline>
          </font>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tab">
          <attribute name="title">
           <string>En cours</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_6">
           <item row="0" column="2">
            <widget class="QLabel" name="abo_type_field">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>21</height>
              </size>
             </property>
             <property name="text">
              <string>-</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLabel" name="abo_validity_label">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>21</height>
              </size>
             </property>
             <property name="text">
              <string>Date d'achat: </string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLabel" name="abo_gear_label">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>21</height>
              </size>
             </property>
             <property name="text">
              <string>MatÃ©riel inclus: </string>
             </property>
            </widget>
           </item>
           <item row="0" column="6">
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="6" column="1" colspan="7">
            <layout class="QHBoxLayout" name="horizontalLayout_10">
             <item>
              <layout class="QGridLayout" name="gridLayout_7">
               <item row="1" column="0">
                <widget class="QPushButton" name="delete_button">
                 <property name="minimumSize">
                  <size>
                   <width>100</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Effacer</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="2">
                <spacer name="horizontalSpacer_3">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item row="1" column="3">
                <widget class="QPushButton" name="validate_button">
                 <property name="minimumSize">
                  <size>
                   <width>100</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Valider</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QPushButton" name="create_button">
                 <property name="minimumSize">
                  <size>
                   <width>100</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>CrÃ©er abo</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item row="2" column="2">
            <widget class="QLabel" name="abo_gear_field">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>21</height>
              </size>
             </property>
             <property name="text">
              <string>-</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1" colspan="7">
            <layout class="QHBoxLayout" name="entrances_layout">
             <item alignment="Qt::AlignHCenter">
              <widget class="QCheckBox" name="entrance_1">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item alignment="Qt::AlignHCenter">
              <widget class="QCheckBox" name="entrance_2">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item alignment="Qt::AlignHCenter">
              <widget class="QCheckBox" name="entrance_3">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item alignment="Qt::AlignHCenter">
              <widget class="QCheckBox" name="entrance_4">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item alignment="Qt::AlignHCenter">
              <widget class="QCheckBox" name="entrance_5">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item alignment="Qt::AlignHCenter">
              <widget class="QCheckBox" name="entrance_6">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item alignment="Qt::AlignHCenter">
              <widget class="QCheckBox" name="entrance_7">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item alignment="Qt::AlignHCenter">
              <widget class="QCheckBox" name="entrance_8">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item alignment="Qt::AlignHCenter">
              <widget class="QCheckBox" name="entrance_9">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item alignment="Qt::AlignHCenter">
              <widget class="QCheckBox" name="entrance_10">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="abo_type_label">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>21</height>
              </size>
             </property>
             <property name="text">
              <string>Type d'abo: </string>
             </property>
            </widget>
           </item>
           <item row="1" column="4">
            <widget class="QDateEdit" name="abo_end_date_field">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>21</height>
              </size>
             </property>
             <property name="buttonSymbols">
              <enum>QAbstractSpinBox::NoButtons</enum>
             </property>
             <property name="calendarPopup">
              <bool>true</bool>
             </property>
             <property name="date">
              <date>
               <year>1900</year>
               <month>1</month>
               <day>1</day>
              </date>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QLabel" name="abo_validity_field">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>21</height>
              </size>
             </property>
             <property name="text">
              <string>-</string>
             </property>
            </widget>
           </item>
           <item row="1" column="5">
            <widget class="QPushButton" name="abo_end_date_button">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>100</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>Changer</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_2">
          <attribute name="title">
           <string>Anciens</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="0" column="0">
            <widget class="QTextBrowser" name="abo_history_browser"/>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>first_name_field</tabstop>
  <tabstop>last_name_field</tabstop>
  <tabstop>phone_field</tabstop>
  <tabstop>email_field</tabstop>
  <tabstop>street_name_field</tabstop>
  <tabstop>street_nr_field</tabstop>
  <tabstop>city_zip_field</tabstop>
  <tabstop>city_name_field</tabstop>
  <tabstop>country_field</tabstop>
  <tabstop>reduced_price_checkbox</tabstop>
  <tabstop>male_button</tabstop>
  <tabstop>female_button</tabstop>
  <tabstop>birthdate_field</tabstop>
  <tabstop>reset_button</tabstop>
  <tabstop>search_button</tabstop>
  <tabstop>save_button</tabstop>
  <tabstop>abo_end_date_field</tabstop>
  <tabstop>abo_end_date_button</tabstop>
  <tabstop>entrance_1</tabstop>
  <tabstop>entrance_2</tabstop>
  <tabstop>entrance_3</tabstop>
  <tabstop>entrance_4</tabstop>
  <tabstop>entrance_5</tabstop>
  <tabstop>entrance_6</tabstop>
  <tabstop>entrance_7</tabstop>
  <tabstop>entrance_8</tabstop>
  <tabstop>entrance_9</tabstop>
  <tabstop>entrance_10</tabstop>
  <tabstop>delete_button</tabstop>
  <tabstop>create_button</tabstop>
  <tabstop>validate_button</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>abo_history_browser</tabstop>
 </tabstops>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
